api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/TestConstants.java

```diff
@@ -35,6 +35,8 @@
 
     public static final LocalDate MAGGIE_DOB = LocalDate.now().minusMonths(6);
     public static final LocalDate LISA_DOB = LocalDate.now().minusMonths(24);
+    public static final LocalDate EXPECTED_DELIVERY_DATE = LocalDate.now().plusMonths(2);
```

Comment:
How about appending `_IN_2_MONTHS` to the name?

Reply:
Yep, will do in the next PR.